doc/api/dns.md

```diff
@@ -172,6 +167,11 @@ dns.lookupService('127.0.0.1', 22, (err, hostname, service) => {
 <!-- YAML
 added: v0.1.27
 -->
+- `hostname` {string}
+- `rrtype` {string}
+- `callback(err, addresses)` {Function}
+  - `err` {Error}
+  - `addresses` {Array}
```

Comment:
I changed it to {string[] | Object[] | Array[string[]] | Object} to cover the possible types.